bin/filter_gtf.py

```diff
@@ -46,6 +53,8 @@ def filter_gtf(fasta: str, gtf_in: str, filtered_gtf_out: str, skip_transcript_i
 
                 if seq_name in seq_names_in_genome:
                     if skip_transcript_id_check or re.search(r'transcript_id "([^"]+)"', line):
+                        if extra_id:
+                            line = line.replace("transcript:", "")
```

Comment:
There's no need to make this conditional. All you did above was add a check on whether the line contained this prefix. Since this replacement won't happen unless it does, that logic is redundant and this line.replace alone will be sufficient. 

But could you amend to something that only replaces `transcript:` in the relevant part of the string, with a regex? I'm a bit nervous about it applying across the whole line.